### DISCUSSION TITLE
Split Example Code Error

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#24851] [Q&A Modules: General] [answered]
I am attempting to run the example code from on this link:

https://mooseframework.inl.gov/moose/source/splits/Split.html#76a55bb6-ab21-4d48-9a17-30276ae31f93

I get the following error when running the example.:

../combined-opt -i Split.i

*** ERROR ***
Invalid option "4" in MooseEnum.  Valid options (not case-sensitive) are "FOURTH EIGHTH".

What do I need to do in order to eliminate this error in order to test the example?

**Comment by GiudGiud** [comment]
Try to write FOURTH instead of 4

**Reply by TLWise** [reply]
That suggestion work like a charm. Thank you. The latest issue with the code as written is the following:
```
mpiexec -n 3 ../combined-opt -i Split.i


*** ERROR ***
/home/twise/projects/moose/modules/combined/FlowAboutCyl/Split.i:106: section 'Splits' does not have an associated "Action".
Did you misspell it?
/home/twise/projects/moose/modules/combined/FlowAboutCyl/Split.i:107: section 'Splits/nuv' does not have an associated "Action".
Did you misspell it?
/home/twise/projects/moose/modules/combined/FlowAboutCyl/Split.i:114: section 'Splits/nu' does not have an associated "Action".
Did you misspell it?
/home/twise/projects/moose/modules/combined/FlowAboutCyl/Split.i:120: section 'Splits/v' does not have an associated "Action".
Did you misspell it?
```

**Reply by GiudGiud** [reply, marked as answer]
ah dont use splits.
Use the Field Split Preconditioner:
https://mooseframework.inl.gov/source/preconditioners/FieldSplitPreconditioner.html

The syntax is almost the same
